src/rcms/fm/app/level1/HCALlevelTwoEventHandler.java

```diff
@@ -1560,9 +1527,6 @@ public void preparingTTSTestModeAction(Object obj) throws UserActionException {
 
       CfgCVSBasePath           = ((StringT)functionManager.getHCALparameterSet().get("HCAL_CFGCVSBASEPATH").getValue()).getString();
       FullCfgScript            = ((StringT)functionManager.getHCALparameterSet().get("HCAL_CFGSCRIPT"     ).getValue()).getString();
-      String TTCciControlSequence = ((StringT)functionManager.getHCALparameterSet().get("HCAL_TTCCICONTROL"  ).getValue()).getString();
-      String LTCControlSequence   = ((StringT)functionManager.getHCALparameterSet().get("HCAL_LTCCONTROL"    ).getValue()).getString();
-      
 
       // configuring all created HCAL applications by means of sending the RunType to the HCAL supervisor
       // KKH: resurrect this if you want to fix prepareTTStestmode
```

Comment:
`TTCciControlSequence` and `LTCControlSequence` are used in the following commented line.
How about commenting these two out?

Reply:
OK sure, I'll add these lines back but commented.